wallet/src/de/schildbach/wallet/transactions/TransactionWrapperHelper.kt

```diff
@@ -21,41 +21,42 @@ import org.bitcoinj.core.Transaction
 import org.bitcoinj.core.TransactionBag
 import org.dash.wallet.common.transactions.TransactionWrapper
 import org.dash.wallet.common.transactions.TransactionWrapperFactory
-import org.slf4j.LoggerFactory
+import java.time.ZoneId
 
 object TransactionWrapperHelper {
-    private val log = LoggerFactory.getLogger(TransactionWrapperHelper::class.java)
     fun wrapTransactions(
         transactions: Set<Transaction?>,
         vararg wrapperFactories: TransactionWrapperFactory
     ): Collection<TransactionWrapper> {
+        wrapperFactories.sortByDescending { it.maxTransactions }
```

Comment:
CoinJoin wrappers will have a much higher hit rate for including transactions, so checking them first